spec/customer_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'customer'
+
+describe Customer do
+
+  let(:takeaway) { double(:takeaway, menu: {:tomato_soup => 5,
+          :steak_and_chips => 15})}
```

Comment:
Style/BlockEndNewline: Expression at 6, 35 should be on its own line.<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.